docs/core/config/statistics.md

```diff
@@ -20,44 +20,19 @@ Dovecot supports gathering statistics from events (see [[link,event_design]]).
 Currently there are no statistics logged by default, and therefore they must
 be explicitly added using the `metric` configuration blocks.
 
-The event filter settings are the only required settings in a metric block.
-The filter specifies which events should be used when calculating the
-statistics for a given metric block.  Event filtering is described in detail
-in [[link,event_filter]].
+The event `filter` setting are the only required settings in a `metric` block.
```

Comment:
* filter can be a link?
 * are .. settings -> is .. setting
 * metric can be a link?

Reply:
Where do the links point to, though?

Reply:
metric_filter and metric settings? Not sure if they exist in this pull request yet.